opm/output/eclipse/Summary.cpp

```diff
@@ -736,13 +733,30 @@ static const std::unordered_map< std::string, ofun > funs = {
     {"BWSAT" , bswat},
     {"BSGAS" , bsgas},
     {"BGSAS" , bsgas},
-
-    /* Misc keywords, the 'extra' handlers will only set the value 0 - with the correct dimension*/
-    {"TCPU" , extra_time}
 };
 
 
-inline std::vector< const Well* > find_wells( const Schedule& schedule,
+static const std::unordered_map< std::string, UnitSystem::measure> misc_units = {
+  {"TCPU"     , UnitSystem::measure::time },
+  {"ELAPSED"  , UnitSystem::measure::time },
+  {"NEWTON"   , UnitSystem::measure::identity },
+  {"NLINERS"  , UnitSystem::measure::identity },
+  {"NLINSMIN" , UnitSystem::measure::identity },
+  {"NLINSMAX" , UnitSystem::measure::identity },
+  {"MLINEARS" , UnitSystem::measure::identity },
+  {"MSUMLINS" , UnitSystem::measure::identity },
+  {"MSUMNEWT" , UnitSystem::measure::identity },
+  {"TIMESTEP" , UnitSystem::measure::time },
+  {"TCPUTS"   , UnitSystem::measure::time },
+  {"TCPUDAY"  , UnitSystem::measure::time },
+  {"STEPTYPE" , UnitSystem::measure::identity },
+  {"TELAPLIN" , UnitSystem::measure::time }
+};
+  
+  
+
+  inline std::vector< const Well* > find_wells( const Schedule& schedule,UnitSystem::measure:: },
```

Comment:
There is some weirdness here, and it won't compile. Removing the UnitSystem... part makes it compile.